treeherder/push_health/push_health.py

```diff
@@ -69,7 +71,8 @@ def get_push_failures(push, option_map):
         test_key = '{}{}'.format(test_name, failure_line.job_guid)
         if test_name and test_key not in tests:
             job = failure_line.job_log.job
-            config = option_map[job.option_collection_hash]
+            config = clean_config(option_map[job.option_collection_hash])
+            platform = clean_platform(job.machine_platform.platform)
```

Comment:
A few of these can also be done in a custom serializer in the [`to_representation`](https://www.django-rest-framework.org/api-guide/relations/#custom-relational-fields) method. [Here's](https://github.com/mozilla/treeherder/blob/master/treeherder/webapp/api/serializers.py#L272)  an example. Or as a `SerializerMethodField`, [like so](https://github.com/mozilla/treeherder/blob/master/treeherder/webapp/api/performance_serializers.py#L214).

Line 73 - the renaming of that field from `failure_line.job_log.job` to `job`- can also be done in a custom serializer.